### ISSUE TITLE
Stuck at installing raminitfs

### ISSUE THREAD

**Issue by geniusled** [clayface/U-boot-MX64-20190430_MX64#3]
Hello, first off thank you very much for providing this repo and files to make these otherwise useless devices work again.  
i was able to install the u-boot but cant get past installing the raminitfs file from the USB drive

i have followed the instructions [HERE](https://github.com/clayface/openwrt/commit/0c9e2a0e5701fe41c9dcbd69865d653a51c7572d) with the files from [this](https://github.com/clayface/U-boot-MX64-20190430_MX64) repo to get openWRT installed.  

my device is the MX64, not A0 variant with locked bootloader and used the (not small) boot file(as confirmed from the instructions).
i had no issues using telnet to log into the device and install/unlock the bootloader.   
i then powered down the device, prepped the USB Drive by formatting it to FAT and loading the openwrt-bcm5862x-generic-meraki_mx64-initramfs-kernel.bin file into the root.   
after booting the device with the usb inserted, i waited for the white flashing light but it never happened. after 30 minutes of nothing(constantly pinged 192.168.1.1 with no response) i power cycled the device and the same thing repeated.

i then dismantled the device and connected to the serial port, i found that the device was not able to find the necessary file form the USB [(see boot log HERE)](https://pastebin.com/w6MjW6dd). i realised that the file in this repo is not named the same as the file the device was looking for (openwrt-bcm5862x-generic-meraki_mx64-initramfs-kernel.bin VS openwrt-bcm53xx-generic-meraki_mx64-initramfs.bin) so i renamed the file and retried with limited success [(see boot log HERE)](https://pastebin.com/Y12CZdik)
i also tried the A0 variant raminitfs with the same errors.

would you have any clues as to what to try next? 

**TLDR:** 
error by using file as is from the repo here:
** Unable to read "openwrt-bcm53xx-generic-meraki_mx64-initramfs.bin" from usb 0:1 **
Wrong Image Format for bootm command
ERROR: can't get kernel image!

error after renaming the file from the repo here:
5194216 bytes read
'##' Booting kernel from FIT Image at 90000000 ...
ERROR: can't get kernel image!

not sure where to go from here

**Comment by geniusled** [comment]
OK so i may have posted prematurely as i found my answer in another closed issue right after posting.
i had not understood that the files in this repo are for the bcm5862 and i have a bcm53xx version.
when i used the [files linked by clayface in this issue](https://github.com/clayface/U-boot-MX64-20190430_MX64/issues/1) everything worked out.
